### PR TITLE
TASK: Refactor absolute namespaces in Flow / Session classes

### PR DESCRIPTION
**Pull request by daniellienert**
Also refactored legacy array syntax.

### CODE REVIEW

File: TYPO3.Flow/Classes/TYPO3/Flow/Session/Session.php [hunk 1]
```diff
@@ -573,14 +578,14 @@ public function close()
      *
      * @param string $reason A reason for destroying the session – used by the LoggingAspect
      * @return void
-     * @throws \TYPO3\Flow\Session\Exception
-     * @throws \TYPO3\Flow\Session\Exception\SessionNotStartedException
+     * @throws Exception
```
Comment (albe) [inline TYPO3.Flow/Classes/TYPO3/Flow/Session/Session.php]: `Session\Exception`

File: TYPO3.Flow/Classes/TYPO3/Flow/Session/SessionInterface.php [hunk 2]
```diff
@@ -155,8 +155,8 @@ public function close();
      *
      * @param string $reason A reason for destroying the session – used by the LoggingAspect
      * @return void
-     * @throws \TYPO3\Flow\Session\Exception
-     * @throws \TYPO3\Flow\Session\Exception\SessionNotStartedException
+     * @throws Exception
```
Comment (albe) [inline TYPO3.Flow/Classes/TYPO3/Flow/Session/SessionInterface.php]: `Session\Exception`

File: TYPO3.Flow/Classes/TYPO3/Flow/Session/TransientSession.php [hunk 3]
```diff
@@ -184,25 +185,25 @@ public function close()
      *
      * @param string $reason A reason for destroying the session – used by the LoggingAspect
      * @return void
-     * @throws \TYPO3\Flow\Session\Exception
+     * @throws Exception
```
Comment (albe) [inline TYPO3.Flow/Classes/TYPO3/Flow/Session/TransientSession.php]: `Session\Exception`

**Comment by daniellienert** [comment]
Thanks @albe for the review. Changed the code accordingly.

**Comment by albe** [comment]
Thx! Looks good now :)

**Pull request merged**